src/embed/main.js

```diff
@@ -79,19 +87,39 @@ function onVideoTap() {
   worldRenderer.videoProxy.play();
   hidePlayButton();
 
+  // TODO(hoch): refactor this.
+  omnitoneDecoder.initialize().then(function () {
+    console.log('initialized');
+    videoElement.play();
+  }, function (error) {
+    console.log(error);
+  });
+
   // Prevent multiple play() calls on the video element.
   document.body.removeEventListener('touchend', onVideoTap);
 }
 
 function onRenderLoad(event) {
   if (event.videoElement) {
+
+    // TODO: activate this only when it's in the 'spatial' audio mode.
```

Comment:
Sure. Can that be a part of index.js file? Setting the property of 'spatialAudio = true' can activate Omnitone.